Documentation/project-docs/developer-guide.md

```diff
@@ -264,7 +263,7 @@ The tests can also be filtered based on xunit trait attributes defined in [`Micr
 Tests marked as `Outerloop` are for scenarios that don't need to run every build. They may take longer than normal tests, cover seldom hit code paths, or require special setup or resources to execute. These tests are excluded by default when testing through `dotnet msbuild` but can be enabled manually by adding the `Outerloop` property e.g.
 
 ```cmd
-build-managed -Outerloop
+build -Outerloop
```

Comment:
This seems to be missing the -test flag

Reply:
It was before as well but your correct I'll add the test flag